### DISCUSSION TITLE
Unable to send a mail.

### DISCUSSION THREAD

**Discussion by mzch** [misskey-dev/misskey#8970] [Q&A]
When sending a test mail, the following error occurrs. Could anyone tell me how to fix this issue?

<img width="1415" alt="スクリーンショット 2022-07-09 0 06 00" src="https://user-images.githubusercontent.com/621178/178020240-0b970adb-6200-449e-bfb1-a4dc18435e35.png">

p.s.
No mail hasn't been sent from my Misskey instance.

**Comment by ThatOneCalculator** [comment]
Most likely your mail config is incorrect.

**Reply by mzch** [reply]
Well, I copied SMTP settings from another app which can send emails, so I think there is no miss configuration...

**Reply by ThatOneCalculator** [reply]
Which email service is it? Gmail has some problems in my testing

**Reply by mzch** [reply]
It's my self hosted mail server. :) Gmail is a very good solution but it has more restrictions than a self hosted server when used by various web apps.

**Reply by ThatOneCalculator** [reply]
I can't seem to reproduce the issue on the latest commit, I don't think I'd be able to provide any further support :(

**Reply by mzch** [reply]
Since my mail server does not accept a connection from Misskey instance, I think this happens inside Misskey.
By the way, Don't you think this issue may relate to #8969?

**Reply by ThatOneCalculator** [reply]
Maybe? I couldn't tell, since there's no logs of what caused the crash.

**Comment by rinsuki** [comment]
when misskey's UI saying only "Invalid param.", actual server response probably contains more info, so can you check server response with DevTools of your web-browser?

(please mask credentials or something like before posts screen shot)

**Reply by rinsuki** [reply]
example:
![image](https://user-images.githubusercontent.com/6533808/178078404-0fcd51be-5e78-421c-accd-5d33ee78de8b.png)

**Reply by mzch** [reply]
I tried it, but...

<img width="1515" alt="スクリーンショット 2022-07-09 17 34 27" src="https://user-images.githubusercontent.com/621178/178098429-fafbd6a9-c4cf-4541-a816-1ffd97c6acc5.png">

Only Misskey returns 400 error.

**Reply by mzch** [reply]
<img width="559" alt="スクリーンショット 2022-07-09 17 37 35" src="https://user-images.githubusercontent.com/621178/178098513-54ec4a3f-949c-4f66-a027-8676b380d1b3.png">

**Reply by rinsuki** [reply]
Please check Network tab

**Reply by mzch** [reply]
Oh, Finally Misskey is unable to boot after upgrading OS to Debian 11.4. See #8978 (in Japanese).
